lib/ears/publisher_retry_handler.rb

```diff
@@ -29,27 +29,33 @@ def run(&block)
       begin
         block.call
       rescue *CONNECTION_ERRORS => e
-        handle_connection_error(e, &block)
+        attempt = handle_connection_error(e, attempt) # rubocop:disable Lint/UselessAssignment
```

Comment:
That's a strange one, the assignment is not useless, but necessary to increase the attempt counter.